ONE_CLICK_OPERATIONS.md

```
@@ -0,0 +1,34 @@
+
+For a release candidate:
+
+```bash
+npm run oneclick:release
+```
+
+This additionally runs the deployment-evidence gate and intentionally fails until genuine Arbitrum Sepolia evidence exists.
+
+## Portfolio model
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)